src/coreclr/src/vm/interopconverter.cpp

```diff
@@ -17,9 +17,72 @@
 #include "runtimecallablewrapper.h"
 #include "cominterfacemarshaler.h"
 #include "binder.h"
+#include <interoplibimports.h> // CreateComInterfaceFlags, CreateObjectFlags
```

Comment:
I would fold this into `interoplibinterface.h`.

Reply:
Reworked to hide those flags in the interoplibinterface implementation.